public/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+
+<html lang="en">
+
+<head>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0"> 
+	<link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css">
+	<link rel="stylesheet" type="text/css" href="styles.css">
+    <script type="text/javascript" src="script.js" defer></script>
+    <title>VelociType</title>
+</head>
+
+<body>
+    <main>
+        <div class="title-container">
+            <h1 aria-live="assertive" tabindex="1">
+                VelociType 
+                <span class="sr-only">
+                    This game takes control of the document focus 
+                    to communicate the progress of the player. To navigate
+                    focus back to the title, press escape. Otherwise, focus
+                    will be placed on the current word to be typed. 
+                </span>
```

Comment:
seems a little strange to be adding instructions to the h1 itself 🤔 is this so that the screen-reader-user definitely hears these instructions?

Reply:
yeah it's for the screen reader reading it out.... We'll try putting the instructions in the div around the h1 instead